extensions/src/main/java/org/pentaho/platform/plugin/services/email/EmailService.java

```diff
@@ -73,23 +96,44 @@ public class EmailService implements IEmailService {
    */
   private File emailConfigFile;
 
+  private static final String REFRESH_TOKEN = "refresh_token";
+  private static final String AUTHORIZATION_CODE = "authorization_code";
+
+  private static final String CLIENT_CREDENTIALS = "client_credentials";
+  private static final String GRANT_TYPE = "grant_type";
+  private static final String SCOPE = "scope";
+  private static final String CLIENT_ID = "client_id";
+  private static final String CLIENT_SECRET = "client_secret";
+  private static final String CODE = "code";
+  private static final String XOAUTH = "XOAUTH2";
+  private static final String REDIRECT_URI = "redirect_uri";
+  private static final String TEST_EMAIL_FAIL = "EmailTester.FAIL";
+  private static final String TEST_EMAIL_SUCCESS = "EmailTester.SUCESS";
+  private static final String OAUTH_COMMAND = "AUTH XOAUTH2 ";
```

Comment:
same comment as https://github.com/pentaho/pentaho-platform/pull/5185/files#r1082864328 variable should be **_OAUTH2_COMMAND_**